playbook/app/pb_kits/playbook/pb_form_group/_form_group.scss

```diff
@@ -0,0 +1,115 @@
+[class^=pb_form_group_kit] {
+  display: inline-flex;
+  flex-direction: row;
+  align-items: flex-end;
+  justify-content: flex-start;
+
+  & [class^=pb_text_input_kit] .text_input_wrapper,
+  & [class^=pb_date_picker_kit] .input_wrapper,
+  & [class^=pb_select] {
+    margin-bottom: 0;
+  }
+
+  & > [class^=pb_text_input_kit]:not(:last-child) {
+    .text_input_wrapper input, [class^=pb_text_input_kit] .text_input_wrapper .text_input {
+      border-bottom-right-radius: 0;
+      border-top-right-radius: 0;
+      border-right-width: 0;
+    }
+  }
+
+  & > [class^=pb_text_input_kit]:not(:first-child) {
+    .text_input_wrapper input, [class^=pb_text_input_kit] .text_input_wrapper .text_input {
+      border-bottom-left-radius: 0;
+      border-top-left-radius: 0;
+    }
+  }
+
+  & > [class^=pb_button_kit]:not(:last-child) {
+    border-bottom-right-radius: 0;
+    border-top-right-radius: 0;
+    min-height: 45px;
+  }
+
+  & > [class^=pb_button_kit]:not(:first-child) {
+    border-bottom-left-radius: 0;
+    border-top-left-radius: 0;
+    border-left-width: 0;
+    min-height: 45px;
+  }
+
+  & > [class^=pb_date_picker_kit]:not(:last-child) {
+    .text_input_wrapper input, [class^=pb_text_input_kit] .text_input_wrapper .flatpickr-wrapper {
+      border-bottom-right-radius: 0;
+      border-top-right-radius: 0;
+      border-right-width: 0;
+    }
+  }
+
+  & > [class^=pb_date_picker_kit]:not(:first-child) {
+    .text_input_wrapper input, [class^=pb_text_input_kit] .text_input_wrapper .flatpickr-wrapper {
+      border-bottom-left-radius: 0;
+      border-top-left-radius: 0;
+    }
+  }
+
+  & > [class^=pb_select]:not(:last-child) {
+    margin-bottom: 0px;
+    .pb_select_kit_wrapper select {
+      border-bottom-right-radius: 0;
+      border-top-right-radius: 0;
+      border-right-width: 0;
+    }
+  }
+
+  & > [class^=pb_select]:not(:first-child) {
+    margin-bottom: 0px;
+    .pb_select_kit_wrapper select {
+      border-bottom-left-radius: 0;
+      border-top-left-radius: 0;
+    }
+  }
+
+  & > [class^=pb_selectable_card_kit]:not(:last-child) label {
+    border-bottom-right-radius: 0;
+    border-top-right-radius: 0;
+    border-right-width: 0;
+  }
+
+  & > [class^=pb_selectable_card_kit]:not(:first-child) label {
+    border-bottom-left-radius: 0;
+    border-top-left-radius: 0;
+  }
+
+  & > [class^=pb_selectable_card_icon_kit]:not(:last-child) [class^=pb_selectable_card_kit] label {
+      border-bottom-right-radius: 0;
+      border-top-right-radius: 0;
+  }
+
+  & > [class^=pb_selectable_card_icon_kit]:not(:first-child) [class^=pb_selectable_card_kit] label {
+    border-bottom-left-radius: 0;
+    border-top-left-radius: 0;
+  }
+
+  & > [class^=pb_selectable_card_icon_kit]:not(:last-child) {
+    [class^=pb_selectable_card_kit] input[type="checkbox"]:not(:checked) ~ label, [class^=pb_selectable_card_kit] input[type="radio"]:not(:checked)~ label {
+      border-right-width: 0;
+    }
+  }
+
```

Comment:
```suggestion
& > [class^=pb_typeahead_kit] {
  min-width: 200px;
}
```

I would suggest adding this to fix the issue with the React version of Typeahead example being too narrow. 

<img width="1904" alt="Screen Shot 2020-11-11 at 10 26 16 AM" src="https://user-images.githubusercontent.com/2293844/98837263-618e8300-2408-11eb-9c67-363a41537a05.png">